staging/nvidia/charts/grafana/templates/configmap-dashboards.yaml

```diff
@@ -1,12 +1,8 @@
-# Generated from 'nodes' from https://raw.githubusercontent.com/coreos/kube-prometheus/master/manifests/grafana-dashboardDefinitions.yaml
-# Do not change in-place! In order to change this file first read following link:
-# https://github.com/helm/charts/tree/master/stable/prometheus-operator/hack
-{{- if .Values.enabled }}
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ template "grafana.fullname" . }}
-  namespace: {{ $.Values.namespace }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
Could you educate me what ".Release.Namespace" represents?

we expect both still in kubeaddons namespace so that they can work with Prometheus and grafana

Reply:
Usually you do not want to explicitly control placement of deployment like this, via value files. Since we are deploying the exporter and grafana charts in the same namespaces of kubeaddons (or where ever prometheus is deployed), this would presumably where you would tell helm to deploy this chart. You can then leverage helm's parameters to deploy these resources in this manner.